etc/afpd/file.c

```diff
@@ -895,6 +895,9 @@ int setfilparams(struct vol *vol,
         }
         switch(  bit ) {
         case FILPBIT_ATTR :
+            /* Preserve file dates after a GS/OS folder copy
+             *
+             * change_mdate = 1; */
             memcpy(&ashort, buf, sizeof( ashort ));
```

Comment:
Both commits should be squashed.

Reply:
Alright, let me put together a new branch that only takes in the AsanteTalk fix.